src/06-extensions.md

```diff
@@ -1,71 +1,8 @@
-# Contributing to and extending the BIDS specification
-
-This page lists some ways that you can get involved with the BIDS community.
-
-## Contributing to BIDS
-
-There are many ways to get involved with the BIDS community!
-
-### The BIDS Starter Kit
-
-If you're new to the BIDS community and you'd like to learn a bit more, we
-recommend checking out the [BIDS Starter Kit](https://github.com/bids-standard/bids-starter-kit/blob/master/README.md).
```

Comment:
are you going to remove the BIDS Starter Kit dropdown menu as well? 
![image](https://user-images.githubusercontent.com/9084751/71898378-726fc780-3159-11ea-9e23-1a65007d7c52.png)

On the website and in your updating PR (https://github.com/bids-standard/bids-website/pull/77), I don't see a reference to the starter kit yet - perhaps it'd be good to add it somewhere.

Reply:
The starter kit is already mentioned on the website under https://bids.neuroimaging.io/getting_started.html. 

Although I did not make the initial section headings - just copied them - I suppose that "getting started" on the main website is more aimed towards _users_ and "get involved" more at _developers/contributors_. The distinction between the two is ideally not very strict, since everyone would contribute one way or another to the BIDS ecosystem (with data, knowledge, insights, skills, technical effort, etc). 

On the bids-specification site on RTD, the starter kit was not only mentioned on https://bids-specification.readthedocs.io/en/stable/06-extensions.html (even though the starter kit does not really deal with how to work on BEPs), but also still features in the top-level menu that appears on the left of the screen.

Reply:
Oh, I now see your 1st question:

> are you going to remove the BIDS Starter Kit dropdown menu as well?

If it is up to me, yes. But I think that should be a separate PR on both sides, so this PR keeps that as is. We first might want to improve the way how the Starter Kit is featured on the main website.  And perhaps even reconsider the current Starter Kit implementation altogether.

Reply:
Okay, that's a good way to move forward.